src/cmdstan-guide/installation.Rmd

```diff
@@ -8,25 +8,30 @@ will install a pre-built version of CmdStan along with the required
 dependencies (i.e. a C++ compiler, a version of Make, and required
 libraries) detailed below under [Source installation].
 
-One can create a new conda environment and install CmdStan with
-the following command.
-
+One can create a new conda environment (named `stan`)
+and install CmdStan with the following command.
 ```
-conda create -n cmdstan -c conda-forge cmdstan
+conda create -n stan -c conda-forge cmdstan
```

Comment:
[optional]
Do you think `stan` might be confused here with our repo?  The repo is dev-facing and this is user-facing, so it should be OK.  I can't think of a better name than `stan` that isn't `cmdstan`.  Probably something like `cs` would be too confusible.

Reply:
I've seen `-env` as a suffix for this type of thing, so it would be `stan-env`. Longer than I would personally use, but pretty clear that it's not a magic word I think. Thoughts?

Reply:
I like `-env` because it'll just reinforce that it's an environment in the docs.  I would probably just use `cmdstan` as you originally did (not in the docs, where it's a bit confusing, but in practice).